testplan/common/utils/comparison.py

```diff
@@ -471,16 +474,32 @@ def _partition(results):
     return lhs_vals, rhs_vals
 
 
-def _cmp_dicts(lhs, rhs, ignore, only, report_mode, value_cmp_func):
+def _cmp_dicts(
+    lhs: Dict,
+    rhs: Dict,
+    ignore: Container,
```

Comment:
Wanted to subscript with ```[Hashable]```, but the linter catches it as unsubsrciptable. It appears to be known issue.